### ISSUE TITLE
Complexes - fails for human

### ISSUE THREAD

**Issue by julie-sullivan** [intermine/intermine#1071]
See #804

```
BUILD FAILED
/local-homes/julie/git/intermine/imbuild/integrate.xml:54: The following error occurred while executing this line:
/local-homes/julie/git/intermine/imbuild/source.xml:330: java.lang.NullPointerException
	at java.util.TreeMap.put(TreeMap.java:556)
	at java.util.TreeSet.add(TreeSet.java:255)
	at org.intermine.objectstore.fastcollections.ObjectStoreFastCollectionsForTranslatorImpl.execute(ObjectStoreFastCollectionsForTranslatorImpl.java:253)
	at org.intermine.dataloader.BatchingFetcher$NoseyObjectStore.execute(BatchingFetcher.java:580)
	at org.intermine.objectstore.query.ResultsBatches.fetchBatchFromObjectStore(ResultsBatches.java:192)
	at org.intermine.objectstore.query.PrefetchManager.doRequest(PrefetchManager.java:241)
	at org.intermine.objectstore.query.PrefetchManager.doRequest(PrefetchManager.java:146)
	at org.intermine.objectstore.query.ResultsBatches.getBatch(ResultsBatches.java:164)
	at org.intermine.objectstore.query.ResultsBatches.getRowsFromBatch(ResultsBatches.java:139)
	at org.intermine.objectstore.query.Results.range(Results.java:197)
	at org.intermine.objectstore.query.SingletonResults.range(SingletonResults.java:65)
	at org.intermine.objectstore.query.Results.get(Results.java:226)
	at org.intermine.objectstore.query.Results$Iter.hasNext(Results.java:378)
	at org.intermine.dataloader.ObjectStoreDataLoader.process(ObjectStoreDataLoader.java:135)
	at org.intermine.dataloader.ObjectStoreDataLoader.process(ObjectStoreDataLoader.java:60)
	at org.intermine.dataloader.ObjectStoreDataLoaderTask.execute(ObjectStoreDataLoaderTask.java:128)
	at org.apache.tools.ant.UnknownElement.execute(UnknownElement.java:292)
	at sun.reflect.GeneratedMethodAccessor4.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.tools.ant.dispatch.DispatchUtils.execute(DispatchUtils.java:106)
	at org.apache.tools.ant.Task.perform(Task.java:348)
	at org.apache.tools.ant.Target.execute(Target.java:435)
	at org.apache.tools.ant.Target.performTasks(Target.java:456)
	at org.apache.tools.ant.Project.executeSortedTargets(Project.java:1393)

```

Everything loads into the items database just fine, so we must have an empty collection somewhere perhaps. We also need a better error msg.

**Comment by julie-sullivan** [comment]
user error.

**Comment by julie-sullivan** [comment]
@rachellyne Take a look at tert-terc_human.xml and tert-rmrp_human.xml. They have the protein O14746 interacting with an enzyme and snoRNA.

**Comment by julie-sullivan** [comment]
Fixed by 40c3ec8c1702a595f99db718f06e789663317bbc

**Comment by rachellyne** [comment]
Yes - that's correct.  The enzyme is a protein.  We changed the interactions so that the interactors could be any type.
